spec/services/linkables_service_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe LinkablesService do
+  describe "#select_options" do
+    it "returns an array of titles with content_ids" do
+      linkable = { "content_id" => SecureRandom.uuid, "internal_name" => "Linkable 1" }
```

Comment:
Could we replace this with the new hash syntax?

Reply:
We need these to be string keys, to match the API response.

Reply:
aha, thanks! I forget the new syntax only applies to symbols